app/assets/stylesheets/active_admin/components/_pagination.scss

```diff
@@ -42,3 +42,14 @@
     padding: 1px 5px;
   }
 }
+
+.comments {
+  .pagination {
+    float: left;
+    margin-bottom: 10px;
```

Comment:
@alex-bogomolov can you please make this `30px`? I'd like to get some more breathing room between the comments and the new comment form. I'll make more improvements later for v2 but with this what's there is good. Thanks! 👍🏻